packages/e2e/test/web-extension/dapp-sdk/features/sendSeveralAssets.ts

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { catchError, combineLatest, tap } from 'rxjs';
+import type { ObservableWallet } from '@cardano-sdk/wallet';
+
+export const sendSeveralAssets = ({
+  connectedWallet,
+  logger
+}: {
+  logger: typeof console;
+  connectedWallet: ObservableWallet;
+}) => {
+  const addressAssetsElement = document.querySelector('#info-several-assets-tokens')!;
+
+  combineLatest([connectedWallet.addresses$, connectedWallet.balance.utxo.available$])
+    .pipe(
+      tap(async ([addresses, availableUtxos]) => {
+        if (!connectedWallet) {
+          return logger.warn('Please connect the wallet first');
+        }
+        if (!availableUtxos.assets || availableUtxos.assets?.size === 0) {
+          return 'No assets';
```

Comment:
Isn't this failing silently? :thinking: 
```suggestion
          throw new Error('No assets')
```

Reply:
that's right, addressed here: 92125b1ab7943a80c9ae997ed30da5489cebec65